### ISSUE TITLE
Pages API should identify the exemplar image

### ISSUE THREAD

**Issue by jhammock** [EOL/tramea#343]
(this ticket may be a duplicate, but I cannot find the original)

This has been requested by HHMI and @hyanwong.

**Comment by hyanwong** [comment]
Yes, thanks. I can't find the original either, so I may not have git issued it, but simply mentioned it somewhere else. When I chatted about it to @JRice the only debate was whether to return JSON like {'exemplar':'true',...} and {'exemplar':'false',...}, or simply to omit 'exemplar':'false' when the object isn't an exemplar. I don't care much either way, although the second option uses marginally less bandwidth, I suppose.

**Comment by arlin** [comment]
The Phylotastic project would also like to have this feature.  Maybe we could get together some day for a discussion on how best to deal with images.
